active-directory-b2c-wpf/App.xaml.cs

```diff
@@ -27,7 +20,7 @@ public partial class App : Application
 
         public static string[] ApiScopes = { "https://fabrikamb2c.onmicrosoft.com/helloapi/demo.read" };
         public static string ApiEndpoint = "https://fabrikamb2chello.azurewebsites.net/hello";
-        private static string AuthorityBase = $"https://{AzureAdB2CHostname}/tfp/{Tenant}/";
+        public static string AuthorityBase = $"https://{AzureAdB2CHostname}/tfp/{Tenant}/";
```

Comment:
I guess we could remove the tpf if we use only .WithAuthority?

Reply:
todo: remove `/tfp/`